puppet/modules/jenkins_job_builder/files/theforeman.org/pipelines/lib/rpm_packaging.groovy

```diff
@@ -6,8 +6,20 @@ def update_build_description_from_packages(packages_to_build) {
     currentBuild.description = build_description
 }
 
+def diff_filter(range, filter, path) {
+    return sh(returnStdout: true, script: "git diff ${range} --name-only --diff-filter=${filter} -- '${path}'").trim()
```

Comment:
One of these days, I think we should make this call structure a method since I have seen us use it quite a few times.